### PR TITLE
Adding the second edition of The State of Lightning by Arcane Research

### PR DESCRIPTION
**Pull request by paulosacramento**
**Preview**

- https://deploy-preview-845--bitcoin-design-site.netlify.app/guide/resources/design-research/#the-state-of-lightning-volume-2

### CODE REVIEW

File: guide/resources/design-research.md [hunk 1]
```diff
@@ -97,6 +97,12 @@ A survey and follow-up interview of 990 bitcoin users to determine bitcoin manag
 
 ## Independent research
 
+#### [The State of Lightning Volume 2](https://arcane.no/research/reports/the-state-of-lightning-volume-2)
+
+_2022 by Arcane research_
+
+In this second edition, Arcane Research delivers valuable insights into the growth of Bitcoin's main payment layer. The report helps quantify and visualize what's happening right now with Lightning. Not only that, it also shows how it is solving real problems for people around the world.
```
Comment (Bosch-0) [inline guide/resources/design-research.md:104]:
```suggestion
In this second edition, Arcane Research delivers valuable insights into the growth of bitcoin's lightning network. The report helps quantify and visualize what's happening right now with lightning. Not only that, it also shows how it is solving real problems for people around the world.
```

**Review by Bosch-0 — CHANGES_REQUESTED** [review]
LGTM, minor nit

File: guide/resources/design-research.md (hunk 1 above)
Comment (paulosacramento) [reply guide/resources/design-research.md:104]: Thanks for the correction.

**Comment by sbddesign** [comment]
It looks like a login is required to view this research document?

**Comment by Bosch-0** [comment]
We discussed in the Jam session last night about avoiding linking users to resources they need to log in for to access such as this one. This goes against our views on having no tracking. We have two options about how we could do this: 

- Download and host these documents our selves
- Link to a third-party pdf (such as https://assets.ctfassets.net/4rilomtvvae4/4Q8jnXMVUDMovAY8hcpYkL/fb65b3290c8dcb23314aacb6f9b19be6/The_State_of_Lightning_Vol_2.pdf for this article) 

Thoughts?

**Comment by GBKS** [comment]
I don't think we should host files whose authors restrict their access. Either we add a note ("requires login") on the page, or we just don't list it.

**Comment by paulosacramento** [comment]
I'm in favor of adding the _requires login_ note.

**Comment by pavlenex** [comment]
> I don't think we should host files whose authors restrict their access. Either we add a note ("requires login") on the page, or we just don't list it.

Agreed, just add a note, a label. I think people would def miss out on things, some of those insights are really helpful and TBH people can just use a fake email, so it's not really a big deal imo.

File: guide/resources/design-research.md [hunk 2]
```diff
@@ -97,6 +97,12 @@ A survey and follow-up interview of 990 bitcoin users to determine bitcoin manag
 
 ## Independent research
 
+#### [The State of Lightning Volume 2](https://arcane.no/research/reports/the-state-of-lightning-volume-2)
+
+_2022 by Arcane research_
```
Comment (GBKS) [inline guide/resources/design-research.md:102]:
```suggestion
_Login required to access. 2022 by Arcane research_
```

**Comment by GBKS** [comment]
Made a suggestion for a login note. There's also a conflict that needs to be resolved, is that something you're comfortable doing, @paulosacramento?

**Comment by paulosacramento** [comment]
> Made a suggestion for a login note. There's also a conflict that needs to be resolved, is that something you're comfortable doing, @paulosacramento?

I tried but could not resolve the conflict. It would be great if you or someone else could solve this.

File: guide/resources/design-research.md (hunk 2 above)
Comment (GBKS) [reply guide/resources/design-research.md:102]: @paulosacramento you'll have to press `Commit suggestion` above if you want to accept the change. `Resolve conversation` only closes the conversation without accepting the change.

**Comment by GBKS** [comment]
@paulosacramento I fixed the conflict.

Did you mean to accept or dismiss the change I suggested for the "Login required" copy?

**Review by paulosacramento — COMMENTED** [review]
I hope i did it right. LOL

**Review by Bosch-0 — APPROVED** [review]

**Pull request merged**